$NetBSD: patch-ab,v 1.3 2002/11/06 19:02:56 atatat Exp $

We are similar enough to OpenBSD to use this, especially since it
makes the graphics/avifile-devel pkg build again.

--- include/SDL_syswm.h.orig	Sun Aug 25 02:21:49 2002
+++ include/SDL_syswm.h	Mon Nov  4 13:00:07 2002
@@ -49,7 +49,7 @@
 #else
 
 /* This is the structure for custom window manager events */
-#if (defined(unix) || defined(__unix__) || defined(_AIX) || defined(__OpenBSD__)) && \
+#if (defined(unix) || defined(__unix__) || defined(_AIX) || defined(__OpenBSD__) || defined(__NetBSD__)) && \
     (!defined(DISABLE_X11) && !defined(__CYGWIN32__) && !defined(ENABLE_NANOX))
  /* AIX is unix, of course, but the native compiler CSet doesn't define unix */
 #include <X11/Xlib.h>
